Src/GraphicLibs/Ncurses/NcursesLibrary.hpp

```diff
@@ -0,0 +1,39 @@
+/*
+** EPITECH PROJECT, 2022
+** arcAAAAAAAAAAAAAAAAade
+** File description:
+** NcursesLibrary
+*/
+
+#ifndef ARCADE_NCURSESLIBRARY_HPP
+#define ARCADE_NCURSESLIBRARY_HPP
+
+#include "../../CommonInterface/include/IGraph.hpp"
+#include <ncurses.h>
+
+/// \brief NCURSES library using int as template value from IGraph
+class NcursesLibrary : public IGraph {
+    public:
+        /// \brief Constructor
+        NcursesLibrary();
```

Comment:
Could be good to know what happen within the Constructor & Destructor. What are u initializing or destroying